debian/postinst

```diff
@@ -31,36 +31,36 @@ if [ "$LSB_RELEASE" != "precise" -a "$LSB_RELEASE" != "trusty" -a "$LSB_RELEASE"
     db_go
 fi
 
+ENGINE_USER=openquake
+
 (if [ -f /run/rabbitmq/pid -o -f /var/lib/rabbitmq/mnesia/rabbit@$(hostname -s).pid ]; then
```

Comment:
We can homogenize this with what we do in RHEL:
`if  ! rabbitmqctl status &> /dev/null; then`